pkg/goGit/goGit.go

```diff
@@ -59,21 +64,23 @@ func UpdateLocalCopies(names []string, config *config.Config) {
 	}
 }
 
-func Clone(name string, config *config.Config) {
-	url := "https://github.com/" + config.OrgaName
+func Clone(name string, config *config.Config, user *config.User) {
+	url, err := url.JoinPath("https://github.com", name+".git")
+	if err != nil {
+		logr.Errorf("[GoGit] failed creating the url: %v\n", err)
+		return
+	}
 
+	auth := buildAuth(config, user)
 	// Clone the repository to the given directory
-	logr.Infof("[GoGit] Cloning the %v repository to %v", url+name+".git", config.OutputPath+name)
+	logr.Infof("[GoGit] Cloning the %v repository to %v", url, config.OutputPath+name)
 	r, err := git.PlainClone(config.OutputPath+name, false, &git.CloneOptions{
```

Comment:
Use a filepath build with `filepath.Join` instead of appending strings

Reply:
fixed in commit 0a1f9ae